M2/Macaulay2/packages/PackageCitations.m2

```diff
@@ -221,6 +227,10 @@ iCite String := S -> (
 -- The cite command
 cite = new Command from (T -> if T === () then iCite "M2" else iCite T)
 
+citeAs = method()
```

Comment:
I don't really like this implementation. Why did you not like having a user defined section in the documentation of the package?

Reply:
That was my first attempt, actually.  It required a fair amount of coaxing to get SimpleDoc to do what I wanted (keep the indentation, don't strip the newlines, etc.)  And in the end, having it defined as one of the documentation keywords felt a bit strange.

In particular, we could theoretically add citation information for *any* documentation node, and not just the package itself.  It also made it so that we wouldn't be able to place the citation information in the main package documentation page where it seemed like it should go -- near the authors, certification info, etc.  All the stuff that's provided by `newPackage`.